### DISCUSSION TITLE
rustdesk-1.2.0-aarch64-unknown-linux-gnu-ubuntu-18.04.deb

### DISCUSSION THREAD

**Discussion by ShuGuangYouXia** [rustdesk/rustdesk#2422] [Q&A]
rustdesk-1.2.0-aarch64-unknown-linux-gnu-ubuntu-18.04.deb There is no simplified Chinese, and the default language is garbled

**Comment by rustdesk** [comment]
I guess you can report to flutter team.

**Reply by ShuGuangYouXia** [reply]
I am an ordinary user and don't understand. If you understand, please give me feedback. Thanks, CPU D2000 UOS kylinOS ARM

**Comment by ShuGuangYouXia** [comment]
cpu d2000   UOS   cpu ft2000  kylinOS

**Comment by ShuGuangYouXia** [comment]
?
